system/db/startingcities.txt

```diff
@@ -5,9 +5,5 @@
 [
 // Klaipeda - West Siauliai Woods
 //---------------------------------------------------------------------------
-{ startingCityId: 1, map: "f_siauliai_west", x: -628, y: 260, z: -1025 },
-
-// Orsha - Lempransa Pond
-//---------------------------------------------------------------------------
-{ startingCityId: 2, map: "f_siauliai_16", x: -2694, y: 116, z: -1663 },
+{ startingCityId: 1, map: "f_siauliai_west", x: -628, y: 260, z: -1025 }
```

Comment:
Do we need to keep this file?

Reply:
Is it not possible to start in different cities anymore?

Reply:
Yeah it's not sadly :( (Why IMC??). Also it occurred to me that the ID for that city is actually `256` and Orsha is `512`. The only way I know is because they changed this before they removed the city option altogether.